integration_tests/runtests.py

```diff
@@ -22,4 +23,16 @@
     #cProfile.run('main(sys.argv[1:])', 'test_profile')
     # arguments are the same as for pytest https://docs.pytest.org/en/latest/usage.html
     # or run pytest -h
-    sys.exit(main(sys.argv[1:]))
+    iteration = 0
+    maxIterEnvName = 'IT_MAX_ITERATIONS'
```

Comment:
Curious why this isn't a command-line argument instead of an environment variable?  We already use custom command-line arguments for other features, so curious why we wouldn't do the same here.  Should be trivial to search the arguments for this option, extract it, and pass the rest to pytest.